features/other_a/page_title.feature

```diff
@@ -31,3 +31,13 @@ Scenario: Page title should change when tags are edited
   When I view the work "New Story"
   Then the page title should include "Harry Potter"
   And the page title should not include "Stargate"
+
+Scenario: Page title should be informative on the adult content notice page
+
+  Given I am logged in as "author"
+  And I post the 2 chapter work "New Story" with fandom "Stargate" with rating "Mature"
+  When I am logged out
+  And I view the work "New Story"
+  Then the page title should include "New Story - author - Stargate"
```

Comment:
Here it would be nice to make sure that we're actually on the "adult content notice" page, because it's only implied in the testing steps. Currently, with just reading the test and knowing that it passed, you couldn't be sure that the intended case is getting tested. So you could add a `I should see "This work could have adult content"` before or after this step and the other page title step to make it clearer

However, I locally checked that the test indeed ends up on the adult content notice page, so this is fine as is. So these comments are optional to address, like I said.